### ISSUE TITLE
[Bug]: CodeLite 17.2.0 crashes on startup on Mac/Ventura/M1

### ISSUE THREAD

**Issue by blockchaindevel** [eranif/codelite#3176]
### What happened?

Installed the latest version of CodeLite (17.2) via Brew on Ventura/M1 Mac

### Version

Other

### Operating system

macOS

### Steps to reproduce

```text
Installed the latest version of CodeLite (17.2) via Brew on Ventura/M1 Mac
Run CodeLite and it quits right away.
```

### Relevant log output

```shell
Translated Report (Full Report Below)
-------------------------------------

Process:               codelite [76675]
Path:                  /Applications/codelite.app/Contents/MacOS/CodeLite
Identifier:            codelite
Version:               ???
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2023-03-20 23:35:27.2568 -0400
OS Version:            macOS 13.2.1 (22D68)
Report Version:        12
Anonymous UUID:        C95DF392-2814-DD63-11B8-40C8B5BE82EB

Sleep/Wake UUID:       71637F3A-63F5-479A-894C-CBA47CFE03DF

Time Awake Since Boot: 1400000 seconds
Time Since Wake:       1240246 seconds

System Integrity Protection: enabled

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000

Termination Reason:    Namespace DYLD, Code 1 Library missing
Library not loaded: /opt/homebrew/*/libjpeg.8.dylib
Referenced from: <FA381AE2-CE78-32CB-9B8F-2353672E8AD2> /Applications/codelite.app/Contents/MacOS/libtiff.5.dylib
Reason: tried: '/opt/homebrew/*/libjpeg.8.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/*/libjpeg.8.dylib' (no such file), '/opt/homebrew/*/libjpeg.8.dylib' (no such file), '/usr/local/lib/libjpeg.8.dylib' (no such file), '/usr/lib/libjpeg.8.dylib' (no such file, not in dyld cache)
(terminated at launch; ignore backtrace)
. . .
```

**Comment by eranif** [comment]
thanks for the detailed error report!. Will look into it

**Comment by eranif** [comment]
Can you try:

```bash
brew update
brew reinstall --cask codelite-official
```

**Comment by eranif** [comment]
Closed with 0b3585771662ce6041edc8afb4702e3b607bc637

**Comment by blockchaindevel** [comment]
That did it! Thanks.
Now if only the debugger would work :)
https://github.com/eranif/codelite/issues/3164
